pkg/apis/v1alpha1/vpcnetworkconfiguration_types.go

```diff
@@ -60,6 +60,8 @@ type VPCInfo struct {
 	Name string `json:"name"`
 	// AVISESubnetPath is the NSX Policy Path for the AVI SE Subnet.
 	AVISESubnetPath string `json:"lbSubnetPath,omitempty"`
+	// NSXLBSPath is the NSX Policy path for the NSX LB Service path.
```

Comment:
Hi, pls refer https://github.com/vmware-tanzu/nsx-operator/pull/629#discussion_r1669572471

Reply:
Checked this comment and we already run `make manifests/make generated` and included the changes in this PR.